libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -619,15 +619,15 @@ const AP_Param::GroupInfo NavEKF3::var_info[] = {
     // @Bitmask: 0:FirstEKF,1:SecondEKF,2:ThirdEKF,3:FourthEKF,4:FifthEKF,5:SixthEKF
```

Comment:
We don't support this many EKFs do we?  Might as well keep the parameter descriptions realistic

Reply:
Its the same issue with the EK3_IMU_MASK parameter. Probably 3 is the maximum at the moment.

Reply:
If we change EK3_GSF_RUN & USE, then we should also change the descriptions for the IMU mask parameters for EK3 and EK2 also.

Reply:
The limit is hardware specific.